ailab/db/finesse/test_queries/__init__.py

```diff
@@ -44,3 +38,54 @@ def get_random_chunk(cursor, schema_version, seed=None):
 
     cursor.execute(query)
     return cursor.fetchall()
+
+
+def get_random_crawl(cursor, schema_version):
+    # Execute the SET commands separately
+    cursor.execute(f'SET SEARCH_PATH TO "{schema_version}", public;')
+
+    query = """
+        WITH random_crawl AS (  
+            SELECT id  
+            FROM crawl  
+            WHERE id IN (
+                SELECT entity_id
+                FROM score
+                WHERE score_type = 'current' AND score = 1
+            )
+            AND id IN (
+                SELECT entity_id
+                FROM score
+                WHERE score_type = 'recency' AND score > 0.5
+            )
+            ORDER BY random()
```

Comment:
I mentioned this in couple requests, but reordering all rows by random() is inefficent; better to pick an offset at random (with limit 1)